src/patternfly/components/Spinner/spinner.scss

```diff
@@ -1,23 +1,34 @@
 // @debug $spinner; // check your variable names located in src/patternfly/sass-utilities/component-namespaces.scss
 
+:root,
 .#{$spinner} {
-  --#{$spinner}--diameter: var(--#{$pf-global}--icon--FontSize--xl);
+  --#{$spinner}--diameter: var(--pf-t--global--icon--size--2xl);
   --#{$spinner}--Width: var(--#{$spinner}--diameter);
   --#{$spinner}--Height: var(--#{$spinner}--diameter);
-  --#{$spinner}--Color: var(--#{$pf-global}--primary-color--100);
+  --#{$spinner}--Color: var(--pf-t--global--icon--color--brand--default);
+  --#{$spinner}--AnimationDuration: 1.4s;
+  --#{$spinner}--AnimationTimingFunction: linear;
+  --#{$spinner}--StrokeWidth: 10;
 
+  // path
+  --#{$spinner}__path--StrokeWidth: var(--#{$spinner}--StrokeWidth);
+  --#{$spinner}__path--AnimationTimingFunction: ease-in-out;
+  
   // sizes
-  --#{$spinner}--m-sm--diameter: var(--#{$pf-global}--icon--FontSize--sm);
-  --#{$spinner}--m-md--diameter: var(--#{$pf-global}--icon--FontSize--md);
-  --#{$spinner}--m-lg--diameter: var(--#{$pf-global}--icon--FontSize--lg);
-  --#{$spinner}--m-xl--diameter: var(--#{$pf-global}--icon--FontSize--xl);
-
+  --#{$spinner}--m-sm--diameter: var(--pf-t--global--icon--size--md);
+  --#{$spinner}--m-md--diameter: var(--pf-t--global--icon--size--lg);
+  --#{$spinner}--m-lg--diameter: var(--pf-t--global--icon--size--xl);
+  --#{$spinner}--m-xl--diameter: var(--pf-t--global--icon--size--2xl);
```

Comment:
Do we still need this modifier class since this is the default size of a spinner?

Reply:
I lean to the side of leaving the modifier for completeness but I wouldn't fight about it.

Reply:
IMO the "inline" spinner should be the default, then you would opt-in to a size variation. It seems odd that the default spinner is so large.